Signal/src/ViewControllers/MediaTileViewController.swift

```diff
@@ -518,19 +549,20 @@ fileprivate protocol MediaGalleryCellDelegate: class {
     func didTapCell(_ cell: MediaGalleryCell, item: MediaGalleryItem)
 }
 
-fileprivate class MediaGalleryLoadingHeader: UICollectionViewCell {
+fileprivate class MediaGalleryStaticHeader: UICollectionViewCell {
 
-    static let reuseIdentifier = "MediaGalleryLoadingHeader"
+    static let reuseIdentifier = "MediaGalleryStaticHeader"
 
     let label = UILabel()
 
     override init(frame: CGRect) {
         super.init(frame: frame)
 
-        // TODO add spinnner, start/stop animating on will/end display
         addSubview(label)
 
-        label.autoCenterInSuperview()
+        label.textAlignment = .center
+        label.numberOfLines = 0
```

Comment:
Whenever I allow wrap, I tend to explicitly specify the lineBreakMode, e.g. word.